src/routes/Routes.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import Welcome from "../pages/Welcome/Welcome";
+import Matches from "../pages/Matches/Matches";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import firebase from "../firebase";
+import MockData from "../assets/Mock Data/MockData";

```

Comment:
Something funky going on here - doesn't like the import on my machine
Also as a naming convention I would prefer if we used kebab-case for any files with more than one word rather than having a space :)

Reply:
corrected :)